src/mbgl/text/glyph.hpp

```diff
@@ -89,6 +89,8 @@ class Shaping {
     WritingModeType writingMode;
     std::size_t lineCount = 0u;
     explicit operator bool() const { return !positionedGlyphs.empty(); }
+    // The y offset *should* be part of the font metadata.
+    static constexpr int32_t yOffset = -17;
```

Comment:
Move the explanation of the −17 value from `getGlyphQuads()` in quads.cpp to this constant’s documentation comment.

Reply:
I think I would keep it as Chris and @ansis wrote it.

Reply:
Do we ever document things in two places? if we were to document here and quads.cpp?